book/plugins/android/single-bundle.md

```diff
@@ -10,6 +10,40 @@ Single Bundle is a plugin that makes the implementation of multiple React Native
 #### Add "Single Bundle Plugin" to the Project Configuration
 Add  `"single bundle plugin"`  and from the available list of versions of `react-native`, `react` and `react-native-zapp-bridge` *(The Applicaster standard bridge for communicating between React Native and the Applicaster SDK)* please select the combination which is supported by your plugins.
 
+```
```

Comment:
Please add before this part, that the following JSON represents the necessary for fields for the Single Bundle Plugin manifest file.

Reply:
👍